src/lib/components/data-table/TagCell.svelte

```
@@ -1,34 +1,54 @@
 		<Tooltip.Root>
 			<Tooltip.Trigger>
-				<span class="block max-w-[25vw] truncate">{formatted}</span>
+				<div class="flex max-w-full items-center gap-1 overflow-hidden">
+					{#each visibleTags as tag}
+						<Badge
+							variant="secondary"
+							class="bg-muted text-muted-foreground max-w-40 shrink-0 border-transparent"
+						>
+							<span class="truncate">{tag}</span>
+						</Badge>
+					{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)